Update tutorials (#68)
lampepfl/progfun-wiki: CheatSheet.md

We copied it and changed or added a few things.
There are certainly a lot of things that can be improved! If you would like to contribute, you have two options:

- Click the "Edit" button on this file on GitHub:
  [https://github.com/lampepfl/progfun-wiki/blob/gh-pages/CheatSheet.md](https://github.com/lampepfl/progfun-wiki/blob/gh-pages/CheatSheet.md)
  You can submit a pull request directly from there without checking out the git repository to your local machine.

- Fork the repository [https://github.com/lampepfl/progfun-wiki](https://github.com/lampepfl/progfun-wiki) and check it out locally. To preview your changes, you need [jekyll](http://jekyllrb.com/). Navigate to your checkout and invoke `jekyll serve`, then open the page [http://localhost:4000/CheatSheet.html](http://localhost:4000/CheatSheet.html).

## Source Files, Classfiles and the JVM

> Scala source code is stored in text files with the extension `.scala`. Typically

Reviewer: ```suggestion
Scala source code is stored in text files with the extension `.scala`. Typically,
```

Scala programmers create one source file for each class, or one source file for

> a class hierarchy: In fact, Scala allows multiple classes and objects to be
> defined in the same source file.  

Reviewer: ```suggestion
a class hierarchy. In Scala, you can define multiple classes and objects
in the same source file.  
```
